goth/configuration.py

```diff
@@ -0,0 +1,245 @@
+"""Defines a class representing `goth` configuration."""
+from pathlib import Path
+from typing import Any, Dict, List, Optional, Type, Union
+
+import yaml
+
+from goth.address import YAGNA_REST_URL, PROXY_HOST
+from goth.runner.container.compose import (
+    ComposeConfig,
+    DEFAULT_COMPOSE_FILE,
+    YagnaBuildEnvironment,
+)
+from goth.runner.container.payment import PaymentIdPool
+from goth.node import node_environment
+from goth.runner.probe import Probe, YagnaContainerConfig
+
+
+class Configuration:
+    """Configuration of a `goth` test network."""
+
+    containers: List[YagnaContainerConfig]
+    """A list of container configurations for nodes in the test network."""
+
+    web_root: Path
```

Comment:
```suggestion
    web_root: Optional[Path]
```

Reply:
At the moment this path is not optional. We pass `Configuration.web_root` as the value of the (non-optional) `web_root_path` parameter to `Runner.__init__()`.